src/components/application_manager/include/application_manager/commands/command_request_impl.h

```diff
@@ -112,8 +112,15 @@ class CommandRequestImpl : public CommandImpl,
  public:
   enum RequestState { kAwaitingHMIResponse = 0, kTimedOut, kCompleted };
 
+  /**
+   * @brief The HashUpdateMode enum defines whether request has to update
+   * hash after its execution is finished
+   */
+  enum HashUpdateMode { kDoHashUpdate, kNoHashUpdate };
```

Comment:
@dev-gh also propose swap this values. It looks more logically when `kSkipHashUpdate = 0` and `kDoHashUpdate = 1`